newton/sim/types.py

```diff
@@ -56,3 +56,11 @@ class ShapeGeometry:
     source: wp.array(dtype=wp.uint64)
     scale: wp.array(dtype=wp.vec3)
     filter: wp.array(dtype=int)
+
+
+# Types of equality constraints
+EQ_CONNECT = wp.constant(0)
+EQ_WELD = wp.constant(1)
+EQ_JOINT = wp.constant(2)
+EQ_FLEX = wp.constant(3)
+EQ_TENDON = wp.constant(4)
```

Comment:
I would remove flex and tendon types for now since these are not implemented and also very mujoco specific.